packages/backup-restore-operator/charts/values.yaml

```diff
@@ -7,14 +7,14 @@ s3:
   enabled: false
   ## credentialSecretName if set, should be the name of the Secret containing AWS credentials.
   ## To use IAM Role, don't set this field
-  credentialSecretName: creds 
+  credentialSecretName: ""
   credentialSecretNamespace: ""
-  region: us-west-2
-  bucketName: rancherbackups
-  folder: base folder
-  endpoint: s3.us-west-2.amazonaws.com
-  endpointCA: base64 encoded CA cert
-  # insecureTLSSkipVerify: optional
+  region: ""
+  bucketName: ""
+  folder: ""
+  endpoint: ""
```

Comment:
If I deploy this as-is with just `s3.enabled=true` as a Helm chart user, what would happen if the endpoint was not changed here? Seems like we would end up with a secret whose stringData has just `endpoint: ""` in it, but no actual failures on chart deploy based on:

https://github.com/rancher/charts/blob/1a5c676c23b49d509eb94f39f3143eb7ed0fd2c0/packages/backup-restore-operator/charts/templates/s3-secret.yaml#L25

Is this a valid configuration to have? If not, we should modify the above line to use the Helm `required` function so that users are forced to pick a value for it if they set `s3.enabled=true`.